django/cantusdb_project/main_app/views/chant.py

```diff
@@ -5,7 +5,7 @@
 import threading
 from django.contrib import messages
 from django.contrib.postgres.search import SearchQuery, SearchRank
-from django.db.models import F, Q, QuerySet, Value
+from django.db.models import F, Q, QuerySet, Value, Prefetch
```

Comment:
Do you use `Prefetch` in this file?

Reply:
I was using it in a previous commit but I since changed it to select_related. Somehow those changes are not being reflected in the "files changed" tab though.

Reply:
yeah, I've had that happen sometimes too...cool!